### PR TITLE
minor UI fixes

### LINKED ISSUE #73: Add link to blog
Add “Blog” link to “About” menu (linking to https://medium.com/@MonarchInit)

### LINKED ISSUE #75: Add "how to cite Monarch" page
The website could use a page about how to cite the project. It could be added to the About menu and also linked or mentioned on https://monarchinitiative.org/page/links.
Relates to https://github.com/monarch-initiative/monarch-app/issues/1191 (I'm not sure if there's a way to add links to other issues?)

### PR DESCRIPTION
**Pull request by nathandunn**
fixes #73 and will fix #75

### CODE REVIEW

**Comment by nathandunn** [comment]
also fixes #74

File: src/components/Navbar.vue [hunk 1]
```diff
@@ -91,6 +95,12 @@
             Link To Us
           </b-dropdown-item>
           -->
+          <b-dropdown-item
+            href="https://medium.com/@MonarchInit"
+            target="_blank"
```
Comment (DoctorBud) [inline src/components/Navbar.vue]: FYI, anytime you use `target="_blank"` you should also add `rel="noopener noreferrer"` for security purposes. cf. https://stackoverflow.com/questions/50709625/link-with-target-blank-and-rel-noopener-noreferrer-still-vulnerable

There's probably a nice way to automate this. I believe I've ensured that any links generated via Markdown have this applied, but I'm not sure if Vue Bootstrap is doing that automatically with `b-dropdown-item`.

Comment (nathandunn) [reply src/components/Navbar.vue]: okay, I've fixed that

File: src/views/Citing.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
+<div
+  class="container-fluid monarch-view monarch-about-view">
+
+#### Goals
```
Comment (DoctorBud) [inline src/views/Citing.md]: Looks like you copied AboutMonarch.md, but didn't delete any of the original content. I think there is content and CSS here that is not needed for your How to Cite text.

**Comment by DoctorBud** [comment]
+1

**Pull request merged**